.github/workflows/php-test-plugins.yml

```diff
@@ -68,20 +73,51 @@ jobs:
       - name: Build assets
         run: npm run build
       - name: Install WordPress
-        run: npm run wp-env start
+        run: |
+          if [ "${{ matrix.coverage }}" == "true" ]; then
+            npm run wp-env start -- --xdebug=develop,coverage
+          else
+            npm run wp-env start
+          fi
       - name: Composer Install
         run: npm run wp-env run tests-cli -- --env-cwd="wp-content/plugins/$(basename $(pwd))" composer install --no-interaction --no-progress
+      - name: Update Composer Dependencies
+        run: composer update --with-all-dependencies --no-interaction --no-progress
+      - name: Install PHPUnit
+        run: |
+          if [ "${{ matrix.php }}" == "7.2" ]; then
+            composer require --dev phpunit/phpunit:${{ matrix.phpunit }} --with-all-dependencies
+          else
+            composer require --dev phpunit/phpunit:${{ matrix.phpunit }}
+          fi
+        if: matrix.phpunit != ''
```

Comment:
Nice, so if the matrix doesn't specify the specific version of PHPUnit then it defaults to whatever is in `composer.json`.

Reply:
Yes, this was the case previously when default PHPUnit version 10 was getting used in all the environment. 😅